helm-chart/renku/templates/network-policies.yaml

```diff
@@ -56,6 +56,14 @@ spec:
           namespaceSelector:
             matchLabels:
               kubernetes.io/metadata.name: {{ .Release.Namespace }}
+        - podSelector:
```

Comment:
In this case Prometheus is not a component of Renku, thus we cannot guarantee that it will be installed alongside Renku, which labels its pod would have and in which namespace it would run.
Luckily there is an easy fix: since the port 9187 is only used for metrics we can allow any pod to connect to it.

Reply:
I modified the policy as suggested.

I tested it in the deployment in the `sean` namespace on the dev cluster. This is what the netpol looks like there:

![image](https://user-images.githubusercontent.com/540360/179991784-7736107d-ee0d-4701-8e44-a1283b97fcfc.png)